server/src/db/schema.ts

```diff
@@ -0,0 +1,415 @@
+import {
+  InferInsertModel,
+  InferSelectModel,
+  relations,
+  sql,
+} from 'drizzle-orm';
+import {
+  primaryKey,
+  integer,
+  real,
+  sqliteTable,
+  text,
+} from 'drizzle-orm/sqlite-core';
+import { createInsertSchema, createSelectSchema } from 'drizzle-zod';
+import { createId } from '@paralleldrive/cuid2';
+
+export const user = sqliteTable('user', {
+  id: text('id')
+    .primaryKey()
+    .$defaultFn(() => createId()),
```

Comment:
do we need this lib ?

Reply:
I tried making ids similar to what we have in mongoose. But if we want to use auto increment plz let me know so I can fix it.

Reply:
We can use ``serial`` for auto-increment. 
For UUID we must have to use third party I believe.